content/_index.html

```diff
@@ -48,12 +48,21 @@ <h1>Participating</h1>
 
     <p>Looking forward to seeing you!</p>
 
-    <h2>Upcoming meetings</h2>
+    <h2>Upcoming posts</h2>
     <ul>
-        <li>2024-03-05 - <a href="/post/2024-02-08_preparation">Kickoff preparation meeting</a></li>
         <li>2024-05-07 - Kickoff meeting</li>
     </ul>
 
+    <h2>Active posts</h2>
```

Comment:
Just to understand the logic, what do you mean by "active post"? Posts that will be updated and that will be placed into "Past posts" once finalized?

Reply:
Might be overkill: `Active posts` == `Recent posts`. I might be overthinking this :)

I would like to have the following types of posts:
- upcoming meetings
- non-EPFL meetings (ETHZ, UK, France?)
- general posts (update on the grant proposal, if somebody wants to present an interesting library/application, ...)
- minutes of past meetings
- past meetings

Any good ideas how to present this in a nice way?

Reply:
Maybe we can then have only the "Upcoming Posts" and "Posts" section.

For the different kinds of posts, maybe we could use tags or categories. All the posts will appear, but you can also sort them by tags/categories depending on your interest.

In the future, when the project gets bigger, some of these categories may need a dedicated page.

Reply:
`Upcoming Posts` -> `Upcoming meetings`?
`Posts` - contain past meetings, minutes, and other posts?

Reply:
Yes, perfect for me!